design/deployment_design.md

```diff
@@ -0,0 +1,79 @@
+#  **Dockerless CLI -** deployment
+
+This document is the design document for the deployment of the Python CLI. 
+
+Currently, our Python CLI can be deployed in the following ways:
+
+1. Install the CLI via the pip Python package manager like so: `pip3 install conjur-client`. **Python is required** on machine if installing the CLI in this way. If installed via pip, the user can then run the CLI like so `conjur-cli --help`
+
+2. Install and run the executable (**Python is** **not** required on the machine)
+
+   The CLI is delivered as a standalone executable, packed with the required runtime and dependencies (excluding a few base linked libraries that are expected on the system). Therefore, there is no need for Python or other prerequisites to be installed on machine. The executable can be fetched by navigating to our [Python CLI release page](https://github.com/cyberark/conjur-api-python3/releases/tag/v0.0.5). 
+   
+   If installed in this manner the CLI can be invoked by, `./python-cli-executable-location variable get ...`
+
+### Design
+
+#### Pip3 package
+
+We make CLI package available to the public by pushing to a public repository called PyPi. We will maintain pushing our the Python CLI to pip3 for both customer's machines have have internet access and for our development work. 
+
+Our Python API is also delivered this way and PyPi is the main installation method for it so this should be retained. The infrastructure is already in-place so this would be at no additional effort.
+
+#### Standalone executable
+
+To install the executable, we will provide tar.gz files for each platform on the [release page](https://github.com/cyberark/conjur-api-python3/releases) of our Github repository. tar.gz is a standard archieve format for macOS and other Linux systems and has been used to deliver software in [another one of our open source projects](https://github.com/cyberark/secretless-broker/releases/tag/v1.7.1).
+
+tar.gz is a better solution than zip because it's complement, `unzip`, does not come packaged in the common distributions (RHEL for example). This would require that users install `unzip` on their machines to use our software which should be avoided. We want to use tar.gz because it is usable without needing any additional software as unzip would demand. Each achieved file will have its respective executable as detailed below.
+
+```bash
+linux/conjur-cli.tar.gz
```

Comment:
@sigalsax lets have no structure of folder in the tar
meaning once i run
tar -xf dawin-conjur-cli.tar.gz
i can simply run 
conjur --help
to see if it is working